packages/codemods/README.md

```diff
@@ -2,10 +2,23 @@
 
 > Codemods for easy migration from [styled components](https://styled-components.com/) and [emotion](https://emotion.sh/docs/introduction).
 
+## Usage
```

Comment:
Should we also update/remove https://compiledcssinjs.com/docs/pkg-cli ?

Reply:
yep will do after merging this PR